include/psa/crypto.h

```diff
@@ -2212,7 +2215,7 @@ static psa_aead_operation_t psa_aead_operation_init(void);
  *    of the non-encrypted additional authenticated data each time.
  * -# Call psa_aead_update() zero, one or more times, passing a fragment
  *    of the message to encrypt each time.
- * -# Call psa_aead_finish().
+ * -# Call psa_aead_finish(psa_aead_encrypt).
```

Comment:
Bad search/replace?

Reply:
Apart from this, the PR is ok to merge as far as I'm concerned. It might not be perfect, but it's a major improvement.

Reply:
Thanks. Not sure how I managed to paste that bit of text here, but it has been removed now.

Reply:
Fixed. @Patater Gilles says this is OK to merge. Can you take a look?